blog/relacja-z-meetupu-hitachi-karate/index.md

```diff
@@ -0,0 +1,84 @@
+
+## Fusion Meetup o Technical Writingu i Karate
+### Informacje o spotkaniu
+
+![](images/meta_prez.png)
+
+12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości z poza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.
+
+### Dlaczego Japonia?
+
+* Pracując w japońskiej firmie, jaką jest Hitachi Energy, mamy okazję obcować z kulturą japońską. 
+* Wiele osób w zespole trenowało wschodnie sztuki walk i lubi oglądać filmy prezentujące mistrzów sztuk walk. 
+* Japońska filozofia kaizen stanowi filar dla praktyk takich jak Lean Management, które stosowane są w Hitachi Energy. 
+* W codziennej pracy technical writerów staramy się stale ulepszać metody pracy, co jest zgodne z japońską zasadą dążenia do doskonałości.
+Niecodzienna fuzja tematów tego meetupu przypadła naszym gościom do gustu.
+
+![](images/patrycja_jap.jpg)
+
+[Patrycja Pyrek](https://www.linkedin.com/in/patrycja-pyrek/), jako znawczyni tematu, wprowadziła nas we frapujące meandry kultury japońskiej, którą poznała z pierwszej ręki, kilkakrotnie odwiedzając Kraj Kwitnącej Wiśni i zgłębiając miejscowe zwyczaje. Słuchacze dowiedzieli się między innymi o tym, dlaczego nie należy wbijać pałeczek w ryż podczas posiłku oraz jak rozumieć szyld z godzinami otwarcia japońskiego sklepu od "23:00 - 25:00" :-)
+
+Arigatō, Patrycja!
+
+
+### Tech writer mistrzem wywierania wpływu
+
+![](images/1694696569715.jpg)
+
+W trakcie Meetupu [Monika Dąbrowska](https://www.linkedin.com/in/monika-klimek1/) opowiedziała o tym, co profesor Cialdini - specjalista na skalę światową w dziedzinie wywierania wpływu - może doradzić pisarzom technicznym, którzy mają problem z uzyskaniem uwag ekspertów domenowych co do opracowanego dokumentu. W codziennych bojach, często zwracamy uwagę na narzędzia, problemy techniczne i organizację pracy, a nie należy zapominać również o budowaniu zdrowych relacji ze współpracownikami. Monika przedstawiła nam perspektywę na mechanizmy rządzące budowaniem relacji między ludźmi. Otrzymaliśmy kilka praktycznych wskazówek, które mogą pomóc podczas codziennej pracy dokumentalisty. 
+Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 
```

Comment:
```suggestion
Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *SME*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 
```

Luźna uwaga o konieczności dodania 's jeśli użyjemy subject matter experta w zdaniu gdzie powinien się po polsku odmienić ;) 
Więc może lepiej go skrócić do SME?

Reply:
SME to staje sie wtedy dosc hermetyczne. staram sie pisac tez z mysla o osobach z zerowa wiedza o technical writingu. najzgrabniej przetlumaczyc na polski. Ekspert Domenowy? :D